oss-cloud-backend/handler.js

```diff
@@ -1,37 +1,33 @@
-"use strict";
+const gitHubApiService = require('./gitHubApiService.js');
+const databaseService = require('./databaseService.js');
 
-var gitHubApiService = require("./gitHubApiService.js");
-var databaseService = require("./databaseService.js");
+module.exports.hello = async (event) => ({
```

Comment:
Let's remove hello function, we don't need it anymore.

Reply:
Removed hello function.